components/typography/metadata/typography-deprecated.yml

```diff
@@ -1,19 +1,181 @@
-id: heading-display
-name: Typography (Deprecated II)
+name: Typography (Deprecated)
```

Comment:
This is basically what we had before in `typography.yml`.

Reply:
Do you want to maybe indicate what was the deprecated V1, V2?

Reply:
There is no version number for the previous implementations of typography, they were both present when we moved to individual versioning. Calling them `legacy` and `deprecated` is the best we can do here, I'm afraid.